dali/base/dadfs.cpp

```diff
@@ -3215,6 +3215,22 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
     virtual bool getAccessedTime(CDateTime &dt) = 0;                            // get date and time last accessed (returns false if not set)
     virtual void setAccessedTime(const CDateTime &dt) = 0;                      // set date and time last accessed
     virtual bool isExternal() const { return external; }
+
+    virtual bool getExpire(int &expireDays)
+    {
+        expireDays = queryAttributes().getPropInt("@expireDays", -1);
+        return true;
```

Comment:
I think it's okay that this function always succeeds (returns -1 if @expiryDays not set),
but no value in it always returning true, it would be better if method was:

```
virtual int getExpireDays()
```

Reply:
Changed